repo_e/capital_letter.py

```diff
@@ -0,0 +1,7 @@
+def capital(strings):
+    Capital_str=[]
+    for i, element in enumerate(strings):
+        if  element==element.capitalize():
+                Capital_str.append(element)
+    return Capital_str
+print(capital(['Krowa', 'swinia', 'Byk', 'Kun']))
```

Comment:
try to write unit tests for you functions. You can use `pytest` for this. Create a directory called `tests` and there put your test files:
```sh
mkdir tests
touch tests/test_capital_letter.py
pytest
```
use the examples that I showed you when creating the recipes API as a reference